templates/data/keda-manager.yaml

```diff
@@ -33,6 +33,8 @@ templates:
           limits:
             limits_memory: 3Gi
             limits_cpu: 2
+          run_if_changed_without_documentation:
```

Comment:
I would suggest renaming it to something related to the `skip_if_only_changed` field. For example:
* `skip_if_only_changed_documentation`

Reply:
done